lib/reactotron-core-client/src/serialize.ts

```diff
@@ -89,6 +93,8 @@ function serialize(source, proxyHack = false) {
           return value
         case "number":
           return value
+        case "bigint":
+          return value.toString()
```

Comment:
@markrickert do you have context on why `toString` works for BigInt but not `toJSON`? just a little confusing that we have to override toJSON but toString works.